etc/testing/circle_tests_inner.sh

```diff
@@ -15,6 +15,8 @@ GOPATH=/root/go
 export GOPATH
 PATH="${GOPATH}/bin:${PATH}"
 export PATH
+TESTFLAGS="-json | tee -a /tmp/results.json | jq -j 'select( .Output != null ) | .Output'"
```

Comment:
So this undoes the JSON, and presents the normal test output?

Reply:
Yeah, it's a little backwards because `-json` actually parses the go-formatted test output and then turns it into JSON, which we turn back into the go-formatted output. Long-term we'll probably switch this and do the JSON (and junit) parsing later.